netbox_librenms_plugin/views/__init__.py

```
@@ -142,7 +156,17 @@
     UpdateDeviceTypeView,
 )
 from .sync.devices import AddDeviceToLibreNMSView, UpdateDeviceLocationView  # noqa: F401
-from .sync.interfaces import DeleteNetBoxInterfacesView, SyncInterfacesView  # noqa: F401
+from .sync.interfaces import (
```

Flake8 (7.3.0)

[error] '.sync.interfaces.DeleteNetBoxInterfacesView' imported but unused

(F401)